src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -422,6 +424,18 @@ class PolicyHandler : public PolicyHandlerInterface,
    */
   uint32_t GetAppIdForSending() const OVERRIDE;
 
+  /**
+   * @brief Add application to PTU queue if don't have application
```

Comment:
@ydementieiev `don't have application ...` -> `no application with the same app id exists`

Reply:
@AByzhynar  update description [c04c115ce](https://github.com/smartdevicelink/sdl_core/commit/c04c115ced5d0ea1dfcc5bb53fc9c01d6d09a998)